### ISSUE TITLE
_WD_GetSession - Multiple Sessions

### ISSUE THREAD

**Issue by seadoggie01** [Danp2/WebDriver#63]
This doesn't appear to work with multiple sessions running. It returns the value of the response from the last session created.

Also, this causes $sSession and $sFuncName to go unused in the function :) Not sure why the commented out code, but I'm guessing this is a bit of a WIP? It claims to return a dictionary object, but returns a string as far as I can tell.

**Comment by Danp2** [comment]
The `Get Session` functionality was added and then removed from the W3C draft spec, so the code is commented until they determine how this should function. See https://github.com/w3c/webdriver/commit/35df53a8a71500e264497f0cbda9f30a94e548cc for details. 

Meanwhile, I temporarily changed the code to return the information that is available. Probably should update the function header with these details.

**Comment by mlipok** [comment]
@seadoggie01 can you please prepare some small reproducer ?

**Comment by seadoggie01** [comment]
@mlipok No. This is from 2 years ago and I hardly remember what I was doing 😁

If I had to guess, the old documentation said something about retrieving a dictionary of sessions and I wanted to use it to close off extra/errored-out sessions. Now, I write sessions to a global ini file in case I need to close them from another script.

I'd be fine with closing this issue.

**Comment by Danp2** [comment]
Nothing to see here until the `Get Session` functionality is reintroduced to the specs.

😝
